MatterControlLib/PartPreviewWindow/SelectedObjectPanel.cs

```diff
@@ -323,6 +323,14 @@ public void EnsureAvailablity()
 
 		private void ShowObjectEditor((IObject3DEditor editor, IObject3D item, string displayName) scopeItem, IObject3D rootSelection, bool allowOperations = true)
 		{
+			// TODO: Some editors make blind assumptions about context when using DragDropData for reference. Remove DragDropData from PublicPropertiesEditor and 
```

Comment:
Null reference see a few times during development due to use of DragDropData which hadn't been initialized in some contexts. Any chance we'd want to revise these usages, passing them a View3DWidget if they require one?

Reply:
Seems reasonable